### ISSUE TITLE
Hypoxic brain injury

### ISSUE THREAD

**Issue by jagcsun** [obophenotype/human-phenotype-ontology#7441]
**Preferred term label:**
Hypoxic brain injury

**Synonyms**

**Definition (free text, please give PubMed ID)**
Brain injury due to low level of oxygen delivery to the brain that result from interruption of blood flow to the brain (PMID: 30725995)

**Parent term (use hpo.jax.org/app)**
Abnormality of brain morphology HP:0012443

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-6670-9157

@pnrobinson

**Comment by pnrobinson** [comment]
This is not a phenotypic feature, it is a pathophysiology (etiology) and as such outside the scope of HPO.
